tooling/cli.js/src/api/dependency-manager/util.ts

```diff
@@ -60,7 +60,9 @@ function padVersion(version: string): string {
 }
 
 function semverLt(first: string, second: string): boolean {
-  return semver.lt(padVersion(first), padVersion(second))
+  return first !== second
+  // When version 1.0.0 is released this code should work again
```

Comment:
can you add `TODO:` here so we don't forget about it

Reply:
good point